frontend/static/js/App.js

```diff
@@ -18,13 +18,21 @@ const imageUploadOptions = {
     progressText: "![Загружаю файл...]()",
     urlText: "![]({filename})",
     errorText: "Ошибка при загрузке файла :(",
+    allowedTypes: [
+        // Images
+        "image/jpeg",
+        "image/png",
+        "image/jpg",
+        "image/gif",
+        // Videos (for animation only)
+        "video/mp4",
+        "video/quicktime" // .mov (macOS' default record format)
+    ],
     extraHeaders: {
         "Accept": "application/json",
     },
     extraParams: {
         code: imageUploadCode,
-        convert_to: "image/jpeg",
```

Comment:
I tried to keep these settings for images only, but inline-attachment doesn't allow this without rewriting part of the plugin. Maybe we should handle this on files server side?

Reply:
These flags have already been removed from the proxy. Its's ok to drop them here too :)